kubernetes/spack/kibana-spack-io/curator/config-maps.yaml

```diff
@@ -0,0 +1,60 @@
+# Adapted from
+# https://gist.githubusercontent.com/HagaiBarel/5dd00a1ca6b1495584c967723092f354/raw/45d53fce4de5ca1ba6506512c7c98a36c45a906d/curator-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: curator-config
+  namespace: spack
+  labels:
+    app: curator
+data:
+  action_file.yml: |-
+    ---
+    # Remember, leave a key empty if there is no value.  None will be a string,
+    # not a Python "NoneType"
+    #
+    # Also remember that all examples have 'disable_action' set to True.  If you
+    # want to use this action as a template, be sure to set this to False after
+    # copying it.
+    actions:
+      1:
+        action: delete_indices
+        description: "Clean up ES by deleting old indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
```

Comment:
Let's apply this action only to the logstash indexes.  The new data from Gitlab shouldn't grow over time in the same way, and I think we should be OK with keeping it all.

```YAML
      # try adding this filter
      - filtertype: pattern
        kind: prefix
        value: logstash-
        exclude:
```

Reply:
👍